addons/forum-post-countdown/addon.json

```
@@ -21,6 +21,14 @@
       "link": "https://scratch.mit.edu/users/mybearworld/"
     }
   ],
+  "settings": [
+    {
+      "name": "When you get the message, automatically retry when possible",
```

Setting descriptions were recently added. Maybe one could be useful here?
```suggestion
      "name": "Retry after countdown",
      "description": "When posting fails because of the 60 second rule, automatically try again when possible."
```